workshops/weather_app/README.md

```diff
@@ -0,0 +1,416 @@
+
+```html
+<script src="script.js"></script>
+``` 
+inside the body tag.
+
+To link external file like `moment.js` we use [`CDNJS`](https://cdnjs.com/) which is an open-source CDN service powered by Cloudflare. 
+
+Add 
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.0/moment.min.js"></script>
+``` 
+inside the `head` tag.
+
+Since we have linked all our files with the `HTML` file, now we can start making a card on which we would display the content.
+
+P.S: For more information about cards, refer to [`Bootstrap`](https://getbootstrap.com/docs/4.0/components/card/), an open source `CSS` framework.
+
+Inside our `<body>` we will add `<div>` with class as `container` which makes our main card and for the content inside we can add another
+`<div>` with class as `content`.
+
+```html
+<div class="container">
+<div class="content">
+```
+
+Then add a heading to our app using `<h1>` .
+```html
+<h1>WEATHER APP</h1>
+```
+
+To input the city from the user we use the `input` tag and declare it with class as `input` and `id` as `input`. 
+
+`class` is used for passing information in `CSS` and `id` is used for `JavaScript`. 
+
+Also, add a `placeholder` which guides the user about the expected value of the input.
+
+```html
+<input id="input" class="input" placeholder="Enter the City Name">
+```
+
+As the user has already input the city name, the `<div>` of `content` should be closed here.
+
+Lastly, we will add `<div>` with class as `main-weather` to display the weather details.
+
+We will be using `<p>` for every detail we display. Hence we declare the weather details with the following ids
+
+```html
+<p id="date">Date </p>
+<p id="city">City </p>
+<p class="temp" id="temp">Temp </p>
+<p id="min-max">Min and Max Temp </p>
+<p id="weather-type">Sunny </p>
+```
+
+Our `HTML` code is done and now we can move to the `CSS` part. At the end, the `HTML` code will look like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.0/moment.min.js"></script>
+	<title>Weather</title>	
+</head>
+
+<body>
+	<div class="container">
+ 		<div class="content">
+    		
+    		<h1>WEATHER APP</h1>
+        	<input id="input" class="input" placeholder="Enter the City Name">
+    	</div>
+
+    <div class="main-weather">
+    	    <p id="date">Date </p>
+            <p id="city">City </p>
+            <p class="temp" id="temp">Temp </p>
+            <p id="min-max">Min and Max Temp </p>
+            <p id="weather-type">Sunny </p>
+    </div>
+</div>
+
+<script src="script.js"></script>
+
+</body>
+</html>
+```
+
+## 4. Improving the design of skeleton using CSS
+
+We will customize the body by giving it background colour, display property, font-size and line-height.
+
+We will set display value as `flex` because Flexbox makes it easier to design a flexible responsive layout without using float or positioning.

```

Comment:
wait what's flexbox? and what's it go to do with flex

Reply:
Initially, I had made something else but then I felt it would be more complex then I removed some code. Would update it!

Reply:
thanks